sender/bms_test.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import bms_data_handler
+
+class test_battery_current_ranges(unittest.TestCase):
+    
```

Comment:
Focus on testing the most crucial functionality - the output format.
In the absence of a test for the output format, any future mistakes will be caught only while integrating with the receiver.